RELEASING.md

```diff
@@ -0,0 +1,315 @@
+- Create an issue for a new release in https://github.com/apache/incubator-stormcrawler/issues  If you do not have permission to do so just ask to be given permissions on the mailing list.
+- Review all of the [issues](https://github.com/apache/incubator-stormcrawler/issues) associated with the release. All issues should be resolved and closed.
+- Any issues assigned to the release that are not complete should be assigned to the next release. Any critical or blocker issues should be resolved on the mailing list. Discuss any issues that you are unsure of on the mailing list.
+
+## Steps for the Release Manager
+
+The following steps need only to be performed once.
+
+- Make sure you have your PGP fingerprint added into https://id.apache.org/
+- Make sure you have your PGP keys password.
+- Add your PGP key to the [KEYS](https://dist.apache.org/repos/dist/release/incubator/stormcrawler/KEYS) file. 
+
+Examples of adding your key to this file:
+
+```
+pgp -kxa <your name> and append it to this file.
+(pgpk -ll <your name> && pgpk -xa <your name>) >> this file.
+(gpg --list-sigs <your name>
+&& gpg --armor --export <your name>) >> this file.
+```
+
+- In a local temp folder, svn checkout the StormCrawler artifacts and update the KEYS file
+- Note: This can only be done by an IPMC member. If you are a committer acting as a release manager, ask an IPMC member to add your key.
+
+```
+svn co https://dist.apache.org/repos/dist/release/incubator/stormcrawler
+svn commit -m "Added Key for <name>" KEYS
+```
+
+- Create a maven `settings.xml` to publish to `repository.apache.org`. An example configuration:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd"
+          xmlns="http://maven.apache.org/SETTINGS/1.1.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <servers>
+        <server>
+            <id>apache.snapshots.https</id>
+            <username>your-asf-ldap</username>
+            <password>your-asf-ldap-password</password>
+        </server>
+        <server>
+            <id>apache.releases.https</id>
+            <username>your-asf-ldap</username>
+            <password>your-asf-ldap-password</password>
+        </server>
+        <server>
+            <id>apache.dist.https</id>
+            <username>your-asf-ldap</username>
+            <password>your-asf-ldap-password</password>
+        </server>
+    </servers>
+    <profiles>
+        <profile>
+            <id>apache-gpg</id>
+            <properties>
+                <gpg.keyname>your-gpg-code-signing-key-fingerprint</gpg.keyname>
+            </properties>
+            <repositories>
+                <repository>
+                    <id>apache.dist.https</id>
+                    <url>https://dist.apache.org/repos/dist</url>
+                </repository>
+            </repositories>
+        </profile>
+    </profiles>
+</settings>
+```
+
```

Comment:
It would be good to add a Note that reminds the release manager to not checkin this settings file.  I have never pushed a release to repository.apache.org and I wonder if there is an ASFInfra request needed to be enabled.

Reply:
The settings.xml is located in ~/.m2 , so it cannot be pushed, but I agree, that we need to make it clear, where to put it.

Nexus access is enabled by default (at least for all projects I am involved with). If the deploy fails, we can open a Jira later.

Might be good to also include the steps needed to do on the Maven Staging repo such as "close" before open a vote.

Reply:
I updated it with the location.